packages/react-instantsearch/src/widgets/SearchBox/SearchBox.js

```diff
@@ -2,6 +2,21 @@ import React, {Component, PropTypes} from 'react';
 
 import themeable from '../../core/themeable';
 import translatable from '../../core/translatable';
+import theme from './SearchBox.css';
+
+const SubmitIcon = () =>
+  <svg xmlns="http://www.w3.org/2000/svg" id="ris-search-icon" viewBox="0 0 40 40">
```

Comment:
We're using `react-svg-loader` on our dashboard react app:

https://github.com/algolia/AlgoliaWeb/blob/master/package.json#L68

Works flawlessly! 👍